test/test.py

```diff
@@ -5,36 +5,71 @@
 ):
     """
+    Evaluate models.
+
     Args:
         model_dirs (list<str>): Comma-separated list of paths to model
                                 directories with checkpoints. These must all
-                                be from the same site.
+                                be from the same site. sites, models, predictors,
+                                and predictors_paths will be ignored if this
+                                parameter is supplied to the function.
+        sites (list<str>): Comma-separated list of site IDs to train on.
+                           Must match the name(s) of the data directories.
+        models (list<str>): Comma-separated list of model names to train.
+                            Options: ['rf', 'ann', 'lasso', 'xgb']
+        predictors (list<str>): Comma-separated list of predictors. Ignored if
+                                predictors_path is provided.
+                                Certain keyword predictors are used to denote
+                                specific sets of predictors:
+                                ['temporal', 'all']
+        predictors_paths (list<str>): Comma-separated list of paths to files
+                                      containing predictor names. See
+                                      predictors/metereological.txt for an
+                                      example.
         split (str): Which split to test on. Options: ['train', 'valid', 'test']
         eval_metrics (list<str>): Metrics to use to evaluate the model(s) on
                                   the split.
 
-    Writes performance metrics to {model_dir}/{split}_results/
-    for each {model_dir}.
+    Writes performance metrics to data/{site}/{split}_results/
     """
+    data_dir = Path("data/")
     if isinstance(model_dirs, str):
         model_dirs = model_dirs.split(",")
-    # # TODO: Add logic for when user passes in sites/models/predictors instead of paths
-    # # (just triple for loop looking for models, print when didn't find one)
-    # if model_dirs is None:
-    #     model_dirs = []
-    #     for model
+    if isinstance(sites, str):
+        sites = sites.split(",")
+    if isinstance(models, str):
+        models = models.split(",")
+
+    predictor_subsets = parse_predictors(predictors_paths, predictors)
+    
+    if model_dirs is None:
+        model_dirs = []
+        for site in sites:
+            save_dir = data_dir / site / "models"
+            for model in models:
+                for predictor_subset in predictor_subsets:
+                    model_dir = save_dir / model / predictor_subset
+                    if len(list(model_dir.glob("*.pkl"))) == 0:
+                        raise ValueError("No models trained for " + 
+                                         f"site={site}, " +
+                                         f"model={model}, " +
+                                         f"predictors={predictor_subset}, "
```

Comment:
**Problem**
Unclear Presentation of Predictors 

**Fix**
`mean_scores['predictors'] = predictor_subsets[predictor_subset]`

Reply:
Done!